### PR TITLE
[Main Course] - Kaggle Jane street

### PR DESCRIPTION
**Pull request by jhqian0909**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Post for kaggle Jane street

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# How Has This Been Tested?

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by jz263 — APPROVED** [review]
LGTM

File: docs/Main_Course/Data_Analytics/2021-09-02-Kaggle-janestreet.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+同样，我们一起来看一下开源的高分代码。
+
+### 3.1 import packages
+
+第一部分照例是导入一堆包。值得一提的是，由于本次数据量较大。代码使用了datatable进行数据读取。datatable据称是一个性能碾压pandas的高效多线程数据处理工具
+
+```python
+import datatable as dtable
+train = dtable.fread('/kaggle/input/jane-street-market-prediction/train.csv').to_pandas()
```
Comment (BulletTech2021) [inline docs/Main_Course/Data_Analytics/2021-09-02-Kaggle-janestreet.md:46]: 我看了一下这个包的一些API，数据处理的API远不如pandas多，所以基本只用于数据加载，加载后还是用pandas进行后续处理，对吧

File: docs/Main_Course/Data_Analytics/2021-09-02-Kaggle-janestreet.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+```
+
+### 3.2 特征工程
+
+由于本次提供的特征仍然是一些不知道含义的匿名特征。代码根据每个特征的分布人为的划分为了4种特征：Linear，Noisy，Negative和Hybrid。
```
Comment (BulletTech2021) [inline docs/Main_Course/Data_Analytics/2021-09-02-Kaggle-janestreet.md:51]: `人为的` -> `人为地`

**Review by BulletTech2021 — APPROVED** [review]

File: docs/Main_Course/Data_Analytics/2021-09-02-Kaggle-janestreet.md (hunk 1 above)
Comment (jhqian0909) [reply docs/Main_Course/Data_Analytics/2021-09-02-Kaggle-janestreet.md:46]: 是的 看上去主要是读进来速度比较快。

File: docs/Main_Course/Data_Analytics/2021-09-02-Kaggle-janestreet.md (hunk 2 above)
Comment (jhqian0909) [reply docs/Main_Course/Data_Analytics/2021-09-02-Kaggle-janestreet.md]: 已修改

**Pull request merged**